migrations/versions/db6b2ef8100f_.py

```diff
@@ -0,0 +1,35 @@
+"""empty message
+
+Revision ID: db6b2ef8100f
+Revises: e3a64b4ca634
```

Comment:
Please change this to d0e7144947d0, so we have no migration script merging.

Reply:
f7c9084